xwiki-platform-tools/xwiki-platform-tool-configuration-resources/src/main/resources/xwiki.properties.vm

```diff
@@ -68,6 +68,11 @@
 #-# more clear.
 # core.hierarchyMode = reference
 
+
+#-# [Since 12.8RC1]
+#-# Define if the reyclebin can be skipped by advanced users when deleting documents.
+# core.recyclebin.skip = false
```

Comment:
Note that this means that the code must be located in `CoreConfiguration.java`. We need to decide if it's the right place.

Reply:
Feels more like something to associate to the refactoring module.

Reply:
Updated to `refactoring.recyclebin.skip`.